### PR TITLE
Updating Default Algorithms 

### PR DESCRIPTION
**Pull request by bwstarr19**
In this update, previously unused algorithms in the default option for the reconstruction file were added. This includes the tikh, tv, and grad algorithm, as well as the fbp algorithm combined with various filters. The no longer necessary version exception was removed as well.

### CODE REVIEW

**Comment by bwstarr19** [comment]
This is an update which includes the previously unused algorithms in the reconstruction file.

**Review by carterbox — CHANGES_REQUESTED** [review]
Please restore tests of the accelerated sirt and mlem functions.

File: benchmarking/project.py [hunk 1]
```diff
@@ -95,12 +111,19 @@ def project(num_angles, width, phantom, trials, noise, emission, output_dir):
...
-            sinogram = np.random.poisson(sinogram / noise) * noise
+            sinogram = np.random.poisson(sinogram / poisson[1]) * poisson[1]
         else:
             norm = np.max(sinogram)
-            sinogram = -np.log(np.random.poisson(np.exp(-sinogram / norm) * noise) / noise) * norm
+            sinogram = -np.log(np.random.poisson(np.exp(-sinogram / norm) *
+                        poisson[1]) / poisson[1]) * norm
+    if salt_pepper[0]:
+        sinogram = tomopy.sim.project.add_salt_pepper(sinogram,
+                    prob=float(salt_pepper[1]), val=float(salt_pepper[2]))
```
Comment (carterbox) [inline benchmarking/project.py]: Cannot cast from None to float.

File: benchmarking/project.py [hunk 2]
```diff
@@ -43,10 +42,26 @@
...
+    default=(False, 0),
+    help='Whether to add poisson noise, and how much.',
+)
+@click.option(
+    '-g',
+    '--guassian',
+    nargs=3,
+    default=(False, 0, 0),
+    help='Whether to add gaussian distortions,'
+    'as well as the mean and standard deviation as well.',
```
Comment (carterbox) [inline benchmarking/project.py]: Make these help strings more specific.

File: benchmarking/project.py [hunk 3]
```diff
@@ -44,24 +44,27 @@
...
-    'as well as the mean and standard deviation as well.',
+    nargs=2,
+    default=(0, 0),
+    help='Whether to add gaussian distortions.'
+    'The first entry is the mean of the guassian distribution,'
+    'the second entry is the standard deviation.',
 )
 @click.option(
     '-s_p',
     '--salt_pepper',
```
Comment (carterbox) [inline benchmarking/project.py]:
```suggestion
    '-s',
    '--salt-pepper',
```
Generally, flags are exactly one letter or whole words separated by `-`.

File: benchmarking/project.py [hunk 4]
```diff
@@ -111,19 +114,21 @@ def project(num_angles, width, phantom, trials, poisson,
     if trials > 1:
         original = np.tile(original, reps=(trials, 1, 1))
         sinogram = np.tile(sinogram, reps=(1, trials, 1))
-    if guassian[0]:
-        sinogram = tomopy.sim.project.add_gaussian(sinogram,
-                    mean=float(guassian[1]), std=float(guassian[2]))
-    if poisson[0]:
+    if guassian[0]>0 or guassian[1]>0:
```
Comment (carterbox) [inline benchmarking/project.py]:
```suggestion
    if guassian[0] > 0 or guassian[1] > 0:
```

**Review by carterbox — APPROVED** [review]
Just fix these style issues, and then it looks ready to merge. Thanks, @bwstarr19!

**Pull request merged**